code/modules/chemistry/Chemistry-Recipes.dm

```diff
@@ -2633,26 +2633,36 @@
 		name = "Sulfuric Acid" // COGWERKS CHEM REVISION PROJECT: This could be Fluorosulfuric Acid instead
 		id = "acid"
 		result = "acid"
-		required_reagents = list("sulfur" = 1, "water" = 1, "oxygen" = 1)
+		required_reagents = list("sulfur" = 1, "oxygen" = 1) //water or steam required, see does_react()
 		result_amount = 2
 		mix_phrase = "The mixture gives off a sharp acidic tang."
 		instant = FALSE
 		reaction_speed = 3
-		temperature_change = 10
+		temperature_change = 4 //changes to 8 in closed containers
+		stateful = TRUE
 
 		on_reaction(var/datum/reagents/holder)
+			if(holder.has_reagent("water"))
+				holder.remove_reagent("water", reaction_speed/2)
```

Comment:
this should probably use `created_volume` rather than `reaction_speed`?

Reply:
true! pushed a fix